conf.yaml

```diff
@@ -627,6 +599,34 @@ contents:
                   -
                     repo:   elasticsearch-rs
                     path:   docs/
+              - title:      Java REST Client (deprecated)
+                prefix:     java-rest
+                current:    *stackcurrent
+                branches:   [ 7.x, 7.15, 7.14, 7.13, 7.12, 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0 ]
```

Comment:
Do we want to continue building the 7.x branch of this book? Should we "freeze" at 7.16 once this change as made, or not build anything newer than 7.15?